src/main/scala/de/zalando/beard/renderer/BeardTemplateRenderer.scala

```diff
@@ -34,6 +36,14 @@ class BeardTemplateRenderer(templateCompiler: TemplateCompiler) {
     renderResult.write(string)
   }
 
+  private def stringRepresentation(value: Any): String = value match {
+    case null => ""
```

Comment:
I'm not sure with this... we might want to configure how rendering works... some might want to see that they have a null value in the template... instead of silently render nothing...